### PR TITLE
style: skybox improvements and refactor

### PR DESCRIPTION
**Pull request by olavra**
## What does this PR change?

- Added sun radiance and intensity to skybox
- Added moon mask for lunar phases
- Added second sun orbiting the first sun
- Tweaked cycle colors and new values for radiance and moon mask
- Modified cloud texture to be channel coded
- (R = Value, G = Mask, B = Highlights)
- Removed dependency of second skybox texture (8k)
- Changed controller slider to use values between 0 an 1
- Changed postprocessing bloom values
- Disabled Lens Flare for all qualities (it wasnt configured)

## How to test the changes?

1. Launch de game
2. See the skybox
3. Open the debug menu and goto skybox
4. Change the value of the slider and set the time
5. Check different times of day and be amazed

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md

### CODE REVIEW

File: Explorer/Assets/DCL/StylizedSkybox/Scripts/SkyboxController.cs [hunk 1]
```diff
@@ -8,6 +8,7 @@ public class SkyboxController : MonoBehaviour
     [HideInInspector]
     public int SecondsInDay = 86400;
     public bool PlayOnStart;
+    public bool StopRefresh = false;
```
Comment (lorux0) [inline Explorer/Assets/DCL/StylizedSkybox/Scripts/SkyboxController.cs:11]: Perhaps it should be a property? You dont care about unity's serialization, right?
Also, i think its better to rename it to `RefreshEnabled` and reverse the conditional, so we can remove negations and its easier to follow.

File: Explorer/Assets/DCL/StylizedSkybox/Textures/horizon_clouds2.png
Comment (lorux0) [inline Explorer/Assets/DCL/StylizedSkybox/Textures/horizon_clouds2.png:1]: Is it OK that this image is so big? Did you check memory usage?

**Review by lorux0 — APPROVED** [review]

File: Explorer/Assets/DCL/StylizedSkybox/Textures/horizon_clouds2.png
Comment (olavra) [reply Explorer/Assets/DCL/StylizedSkybox/Textures/horizon_clouds2.png:1]: thats the original texture provided by the art team

File: Explorer/Assets/DCL/StylizedSkybox/Scripts/SkyboxController.cs (hunk 1 above)
Comment (olavra) [reply Explorer/Assets/DCL/StylizedSkybox/Scripts/SkyboxController.cs:11]: this is only for shibu to be able to edit. I need to refactor the whole thing to be a system in the future

**Review by anicalbano — APPROVED** [review]
💯 Amazed

https://github.com/decentraland/unity-explorer/assets/139157886/007bf36b-4d31-4ae5-b994-3aa62f69e525

**Pull request merged**